tests/unittests/config/test_cc_resizefs.py

```diff
@@ -485,7 +485,24 @@ def test_resize_btrfs_mount_is_rw_has_queue(
         """Queue the resize request if btrfs >= 5.10"""
         m_is_rw.return_value = True
         m_is_dir.return_value = True
-        m_subp.return_value = ("btrfs-progs v5.10 \n", "")
+        m_subp.return_value = SubpResult("btrfs-progs v5.10 \n", "")
+        self.assertEqual(
+            ("btrfs", "filesystem", "resize", "--enqueue", "max", "/"),
+            _resize_btrfs("/", "/dev/sda1"),
+        )
+
+    @mock.patch("cloudinit.util.mount_is_read_write")
+    @mock.patch("cloudinit.config.cc_resizefs.os.path.isdir")
+    @mock.patch("cloudinit.subp.subp")
+    def test_resize_btrfs_version(self, m_subp, m_is_dir, m_is_rw):
+        """Queue the resize request if btrfs >= 5.10"""
```

Comment:
In running `lxc launch images:fedora/40/cloud` I can see btrfs 6.9.2 which also retains the original `btrfs --version` behavior. I think it's just 6.10 that we have to address here not 5.10.
```diff
--- a/tests/unittests/config/test_cc_resizefs.py
+++ b/tests/unittests/config/test_cc_resizefs.py
@@ -495,11 +495,11 @@ class TestMaybeGetDevicePathAsWritableBlock(CiTestCase):
     @mock.patch("cloudinit.config.cc_resizefs.os.path.isdir")
     @mock.patch("cloudinit.subp.subp")
     def test_resize_btrfs_version(self, m_subp, m_is_dir, m_is_rw):
-        """Queue the resize request if btrfs >= 5.10"""
+        """Queue the resize request if btrfs >= 6.10"""
         m_is_rw.return_value = True
         m_is_dir.return_value = True
         m_subp.return_value = SubpResult(
-            "btrfs-progs v5.10 \n\n-EXPERIMENTAL -INJECT -STATIC +LZO +ZSTD "
+            "btrfs-progs v6.10 \n\n-EXPERIMENTAL -INJECT -STATIC +LZO +ZSTD "
             "+UDEV +FSVERITY +ZONED CRYPTO=libgcrypt",
             "",
         )
```